### PR TITLE
Minor cosmetics and updates

### PR DESCRIPTION
**Pull request by bruskiza**

### CODE REVIEW

**Comment by bruskiza** [comment]
@miltontony ready for review again.

**Comment by bruskiza** [comment]
@hodgestar @smn ^^^

File: docs/cheatsheet.rst [hunk 1]
```diff
@@ -114,10 +129,14 @@ The steps are as follows:
...
   <enter text>
 
-2. Start a new feature with hubflow named: issue-<issue # you created in step 1>-<description of work>
+2. Start a new feature with hubflow named: :code:`issue-<issue # you created in step 1>-<description of work>`
 ::
   hub hf feature start issue-1-going-to-write-some-code
+
 3. Write code
+
+This is where the actual magic happens.
```
Comment (hodgestar) [inline docs/cheatsheet.rst:138]: I would have thought this line would need to be indented? (but I don't have a good sense of the overall layout).

File: docs/conf.py [hunk 2]
```diff
@@ -41,7 +42,7 @@
 
 # General information about the project.
 project = u'Praekelt Ways of Working'
-copyright = u'2013, Praekelt Foundation devs and individual contributors'
+copyright = '%d, Praekelt Foundation developers and individual contributors' % date.today().year
```
Comment (hodgestar) [inline docs/conf.py]: We should probably just remove "developers" here -- it's copyright the organization and individual contributors.

Not sure about automatically setting the copyright date, but maybe lets go with this unless anyone complains?

**Comment by hodgestar** [comment]
Minor comments, looks good!

File: docs/cheatsheet.rst (hunk 1 above)
Comment (bruskiza) [inline docs/cheatsheet.rst:138]: cheatsheet_ _praekelt_ways_of_working_1_documentation

@hodgestar i think it looks better unindented.

Comment (hodgestar) [inline docs/cheatsheet.rst:138]: I meant just the "This is where the actual magic happens" bit. If it's
already indented to the same level as "Write code", lets leave it as
is.

**Comment by bruskiza** [comment]
@hodgestar @smn @miltontony: ready for re-review

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**